cmake-tests/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ if(NOT CMAKE_CXX_COMPILER MATCHES ".*hcc")
 endif()
 
 add_executable(cmake-test cmake-test.cpp)
+add_dependencies(cmake-test clang_links rocdl_links)
```

Comment:
This won't work in a standalone build, and this is designed to work both standalone and in the project.